discovery/manager.go

```diff
@@ -140,6 +141,19 @@ func Name(n string) func(*Manager) {
 	}
 }
 
+// MemberRollController creates the service mesh member roll controller
+func MemberRollController(name, namespace string, resync time.Duration) func(*Manager) {
```

Comment:
Nit: I think this should be called `MemberRollConfig` or similar, because it really just sets a few values on the `Manager`. Comment should be changed, too (`applies the MemberRollController-specific configuration on the Manager` or similar)

Reply:
I updated the comment but have left the function name the same

Reply:
That'll do, thanks